Xamarin.Essentials/Contacts/Contacts.shared.cs

```diff
@@ -12,12 +11,23 @@ public static async Task<Contact> PickContactAsync()
             // iOS does not require permissions for the picker
             if (DeviceInfo.Platform != DevicePlatform.iOS)
                 await Permissions.EnsureGrantedAsync<Permissions.ContactsRead>();
-
             return await PlatformPickContactAsync();
         }
+
+        public static Task<IEnumerable<Contact>> GetAllAsync(CancellationToken cancellationToken = default)
+            => PlatformGetAllAsync(cancellationToken);
+    }
+
+    public enum ContactPhoneType
+    {
+        Unknown = 0,
+        Personal = 1,
+        Work = 2,
+        Main = 3,
+        Mobile = 4
     }
 
-    public enum ContactType
+    public enum ContactEmailType
```

Comment:
As discussed in the comments, we can remove this and the properties on the type for now as we decide on how to actually implement this properly. This is mainly because each platform is very different, and some platforms even support arbitrary user values.

This may mean we need a more complex type if we want to support this. 

We also probably need an issue for this.